packages/contracts/package.json

```diff
@@ -9,10 +9,11 @@
     "devnode": "mud devnode",
     "test": "mud test",
     "types": "mud types",
-    "dev": "yarn tablegen && yarn deploy",
+    "dev": "yarn tablegen && yarn worldgen && yarn deploy",
     "deploy": "mud deploy-v2 --clean",
     "lint": "yarn prettier && yarn solhint",
     "tablegen": "mud tablegen",
+    "worldgen": "mud worldgen",
```

Comment:
not blocking here but we should work towards collapsing all of this into a single `mud codegen` command that does all the things